release_notes/Service_Fabric_ReleaseNotes_90CU3.md

```diff
@@ -0,0 +1,83 @@
+# Microsoft Azure Service Fabric 9.0 Cumulative Update 3.0 Release Notes
+
+This release includes bug fixes as described in this documents.
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 9.0.1103.1 <br> 9.0.1103.1 <br> 9.0.1107.9590 |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 9.0.1107.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.0.1107  <br> 9.0.1107 <br> 9.0.1107 <br> 9.0.1107 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Contents 
+
+Microsoft Azure Service Fabric 9.0 Cumulative Update 3.0 Release Notes
+
+* [Breaking Changes](#breaking-changes)
+* [Key Announcements](#key-announcements)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+## Breaking Changes
+Customers running Azure Service Fabric Linux Clusters will not be able to scale out or reimage (manual or auto via VMSS OS Image Update) nodes or create new Linux clusters without upgrading to this version of Service Fabric (9.0.1086.1). Also, if your application requires JDK, then you will need to install a JDK of your choice since SF will not install the JDK as part of the SF installation.
+Starting 8/19/2022, the following versions will be deprecated and will not be available on Linux for new cluster creation or node reimage/scale out operations:<br>
+* 9.0 CU2/CU1/RTO
+* 8.2 CU4/CU3/CU2.1/CU2/CU1/RTO<br>
+
+**Details:** Service Fabric Runtime (for Linux) placed a dependency on Zulu for Azure build of OpenJDK which is no longer supported as announced in their dev [blog](https://devblogs.microsoft.com/java/end-of-updates-support-and-availability-of-zulu-for-azure/). Since the creation of new Linux clusters or scale-out/reimage of nodes required all the dependencies to be installed, these workflows broke when the dependency became unavailable.<br>
+Beginning with this release, Service Fabric Runtime will no longer install any JDK dependency which will ensure that customer can scale out, reimage, or create new Linux clusters. If your application needs JDK, please utilize an alternate mechanism to provision a JDK. To get the latest Service Fabric Runtime releases with this change, follow the upgrade documentation. If you depend on a JDK, please be sure to install one of your choice.<br>
```

Comment:
"Beginning with this release . . . " - which one is "this" release?? When as was Breaking Change first instroduced?

Reply:
echo above, is it really beginning with this release? If we already introduced the change then we should just reiterate one more time that we rolled out a change already and will no longer..